pkgs/development/interpreters/python/cpython/default.nix

```
@@ -299,6 +299,13 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
   ] ++ optionals (pythonOlder "3.12") [
     # https://github.com/python/cpython/issues/90656
     ./loongarch-support.patch
+  ] ++ optionals (pythonAtLeast "3.11" && pythonOlder "3.12") [
+    # fix failing tests with openssl >= 3.4
+    # https://github.com/python/cpython/pull/127361
+    (fetchpatch {
+      url = "https://github.com/python/cpython/commit/11e0523eb363b7def4bc64d24a04e88d8670a691.patch";
```

```suggestion
      url = "https://github.com/python/cpython/commit/f4b31edf2d9d72878dab1f66a36913b5bcc848ec.patch";
```
we should probably take the final commit that got merged